tools/src/test/java/org/apache/pdfbox/tools/TestAddImage.java

```diff
@@ -49,9 +50,7 @@ public void testMain() throws IOException {
     }
 
 
-    public static void main( String[] args )
-    {
-        String[] arg = {AddImage.class.getName() };
-        junit.textui.TestRunner.main( arg );
```

Comment:
Are there any other tests?
I am not able to view the generated pdf file. Will you attache the actual file instead of the path?

Reply:
This test already does exactly that. 
The PDF it uses has text included before-hand